datasets/modeled/industry/dataset.json5

```diff
@@ -12,9 +12,9 @@
     "Ikenna Okeke",
   ],
   origin_project: "IEF Year 2 Task 2.1",
-  origin_date: "2025-04-19",
-  origin_version: "0.1.0",
-  source: "LBNL, ORNL, NREL",
+  data_source_date: "2025-04-19",
+  data_source_version: "0.1.0",
+  data_source_doi_url: "LBNL, ORNL, NREL",
```

Comment:
I'd delete these and put the contributing labs in the description (if they're not already there).